client/my-sites/plan-features/header.jsx

```diff
@@ -248,10 +249,22 @@ export class PlanFeaturesHeader extends Component {
 			return translate( `Save %(discountRate)s%% by paying annually`, { args: { discountRate } } );
 		}
 
-		if ( ( isInSignup || isLoggedInMonthlyPricing ) && ! isMonthlyPlan ) {
+		if (
+			( isInSignup || isLoggedInMonthlyPricing ) &&
+			! isMonthlyPlan &&
+			planMatches( planType, { group: GROUP_WPCOM, term: TERM_ANNUALLY } )
+		) {
 			return translate( 'billed annually' );
 		}
 
+		if (
+			( isInSignup || isLoggedInMonthlyPricing ) &&
+			! isMonthlyPlan &&
+			planMatches( planType, { group: GROUP_WPCOM, term: TERM_BIENNIALLY } )
+		) {
+			return translate( 'billed every two years' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( '/month, billed every two years' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/test/biyearly-text)